### ISSUE TITLE
Possible mistake in "Advanced Graphics" chapter

### ISSUE THREAD

**Issue by racarate** [openframeworks/ofBook#174]
So, I think this is incorrect:

```
ofTranslate(20,20);
ofRotate(45);
ofRect(20,20,20,20);
This draws a square rotated 45 degrees around it's top-left corner.
```
If the parameters to ofRect were instead (0, 0, 20, 20) it would be drawn rotated around its top left corner.  As it is, the square is rotated about a point 20 units up and 20 units left of its top-left corner.

Also, minor nitpick but "I can't do use translate/rotate/scale" sounds weird!

**Comment by arturoc** [comment]
you are correct. that kind of proves the point of that part though :) translate/rotate/scale are hard. and yes the sentence should be "I can't use translate/rotate/scale"

can you send a PR? 

thanks!
